### PR TITLE
Po update

### PR DESCRIPTION
**Pull request by ignatenkobrain**

### CODE REVIEW

**Comment by ignatenkobrain** [comment]
@j-mracek since @jrohel is on vaction and we need this ASAP, assigning to you :P

File: po/zanata.xml [hunk 1]
```diff
@@ -4,101 +4,4 @@
   <project>dnf-plugins-core</project>
   <project-version>master</project-version>
   <project-type>gettext</project-type>
-  <locales>
```
Comment (j-mracek) [inline po/zanata.xml]: @ignatenkobrain By that way do you get translation for all available languages?

**Comment by j-mracek** [comment]
Because I cannot test the functionality please if it works merge it.

File: po/zanata.xml (hunk 1 above)
Comment (ignatenkobrain) [inline po/zanata.xml]: yes, now I get translations for **all** languages instead of specified ones.

**Pull request merged**